docs/production/observability/newrelic.mdx

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
To add the doc to navigation bar, you'll need to add the doc to this file: https://github.com/mage-ai/mage-ai/blob/master/docs/mint.json#L409

Reply:
Done